.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: scala
 sudo: false
 scala:
-  - 2.10.4
```

Comment:
we shouldn't drop scala 2.10 support. Twitter (and other big companies take a long time to upgrade such things). We need our tests to run on the version we use.